src/Symfony/Component/String/Inflector/EnglishInflector.php

```diff
@@ -291,6 +291,9 @@ final class EnglishInflector implements InflectorInterface
         // circuses (circus)
         ['suc', 3, true, true, 'cuses'],
 
+        // campus (campuses)
```

Comment:
I'm not sure, but previous lines are in this way
```suggestion
        // campuses (campus)
```

Reply:
Done ty :)